oak_runtime/src/proxy.rs

```diff
@@ -150,7 +150,9 @@ impl RuntimeProxy {
             self.channel_create("Initial", &Label::public_untrusted())?;
         debug!(
             "{:?}: created initial channel ({}, {})",
-            self.node_name, write_handle, read_handle,
+            NodeInfo::construct_debug_id(&self.node_name, self.node_id),
```

Comment:
Would it make sense to instead have a method to fetch the channel object from the Runtime, and call get_debug_id on it, rather than duplicate this logic here in various places? I am not sure whether it is possible, but worth looking into

Even if not, it seems that this may just be encapsulated as a `get_debug_id` method on this RuntimeProxy object itself, or is there any reason for not doing that?

Reply:
Would not `self.runtime.get_node_debug_id(self.node_id)` work here?  (maybe it is not yet added to `node_infos`?)

I think having a `get_debug_id` function here makes more sense, especially since the logic in `get_debug_id` is very simple, and does not use anything from `NodeInfo`.

Reply:
That line happens before the node is created & registered with the runtime (via `self.node_create`), hence we can't fetch it yet.

However I'm added a get_debug_id method to the proxy. :)